autopart-hibernation.ks.in

```diff
@@ -0,0 +1,77 @@
+#test name: autopart-nohome
+%ksappend repos/default.ks
+network --bootproto=dhcp
+
+bootloader --timeout=1
+zerombr
+clearpart --all --initlabel
+
+# Include the generated autopart command.
+%include /tmp/ksinclude
+
+keyboard us
+lang en
+timezone America/New_York
+rootpw qweqwe
+shutdown
+
+%packages
+%end
+
+%pre
+# Generate the autopart command.
+autopart="autopart"
+if [ "@KSTEST_OS_NAME@" == "fedora" ]; then
+    # --hibernation should work even with --noswap
```

Comment:
[pykickstart](https://pykickstart.readthedocs.io/en/latest/kickstart-docs.html#autopart) documentation says something different:

_Only one of --noswap and --hibernation can be specified._

Reply:
Yeah, it might be better if pykickstart reported an error in this case, so we don't have to test it.

Reply:
Thanks for noticing this. I got a bit confused there. I originally wrote the code in Anaconda to handle both of them but we later decided against it (enforced in pykickstart).